co-simulation/fed/mosaic-infrastructure/pom.xml

```diff
@@ -25,6 +25,13 @@
       <artifactId>mosaic-objects</artifactId>
       <version>${mosaic.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.mosaic</groupId>
+      <artifactId>mosaic-objects</artifactId>
+      <version>${mosaic.version}</version>
+      <type>test-jar</type>
```

Comment:
This package is already included in the infrastructure ambassador pom as a dependency but I had to add the test-jar to gain access to Rules used to initialize singletons for unit testing. See GeoProjectionRule and IpResolverRule in the InfrastructureMessageAdapterTest.